scripts/build.ps1

```diff
@@ -405,8 +405,8 @@ function CMake-Generate {
     if (!$DisableLogs) {
         $Arguments += " -DQUIC_ENABLE_LOGGING=on"
     }
-    if ($StdoutLogs) {
-        $Arguments += " -DQUIC_ENABLE_LOGGING=on -DQUIC_LOGGING_TYPE=stdout"
+    if ($LoggingType) {
+        $Arguments += " -DQUIC_ENABLE_LOGGING=on -DQUIC_LOGGING_TYPE=" + $LoggingType
```

Comment:
Let's put the `$LoggingType` check in the `if (!$DisableLogs)` section. Then you don't need to set `QUIC_ENABLE_LOGGING` just for the type, which makes things confusing if somehow `DisableLogs` is set too.